### PR TITLE
Add /run/user/$UID to possible IPC Paths

### PR DESCRIPTION
**Pull request by SammCheese**
After a bit of research, i've found out that Discord added their discord-ipc-? file to that path for me.
This PR should fix rich presences not appearing for some people

### CODE REVIEW

File: winediscordipcbridge-steam.sh [hunk 1]
```diff
@@ -23,6 +23,7 @@ DISCORD_IPC_PATHS=(
     "$TEMP_PATH/app/com.discordapp.Discord"
     "$TEMP_PATH/snap.discord-canary"
     "$TEMP_PATH/snap.discord"
+    "run/user/$UID"
```
Comment (0e4ef622) [inline winediscordipcbridge-steam.sh:26]: Is this supposed to be a relative path or is it missing a leading slash?

**Review by 0e4ef622 — CHANGES_REQUESTED** [review]
You also need to make a similar change in main.c, although surprising to hear it worked without that.

(Usually `/run/user/$UID` is what `$XDG_RUNTIME_DIR` gets set to so maybe the bridge sees that but the script doesn't?)

File: winediscordipcbridge-steam.sh (hunk 1 above)
Comment (SammCheese) [reply winediscordipcbridge-steam.sh]: Missing Slash, resolved

**Comment by SammCheese** [comment]
Main.c does not need to be edited since the full path (/run/user/$UID/discord-ipc-X) fits the pre-existing ["%s/discord-ipc-%d"](https://github.com/0e4ef622/wine-discord-ipc-bridge/blob/9d564182d42413b01c21bc7ef501a8ee2334ae7b/main.c#L237)

**Review by 0e4ef622 — APPROVED** [review]

**Comment by 0e4ef622** [comment]
Right and the script also uses `$XDG_RUNTIME_DIR` so I'm just curious why this is even necessary.

**Comment by SammCheese** [comment]
>Right and the script also uses $XDG_RUNTIME_DIR so I'm just curious why this is even necessary.

So am i to be honest, but without that change it just doesnt work.

**Pull request merged**